tests/models/test_lisflood.py

```diff
@@ -0,0 +1,111 @@
+import shutil
+from pathlib import Path
+from unittest.mock import patch
+
+import pytest
+import xarray
+
+from esmvalcore.experimental.recipe_output import DataFile
+from grpc4bmi.bmi_client_singularity import BmiClientSingularity
+
+from ewatercycle import CFG
+from ewatercycle.forcing.forcing_data import ForcingData
+from ewatercycle.parametersetdb.datafiles import SubversionCopier
+from ewatercycle.models.lisflood import Lisflood, LisfloodParameterSet
+
+
+@pytest.fixture
```

Comment:
For my understanding, could you explain what we are really testing here? Since we're mostly testing against mocks and fixtures..

Reply:
@sverhoeven could you please reply to this question? thanks.

Reply:
The only thing not real is the bmi object to make tests quick. This test exercises setup() method, but does not check content of config file.

As we switched from unit testing against to lisflood use case (https://github.com/ec-jrc/lisflood-usecases/tree/master/LF_lat_lon_UseCase) to using the notebook for testing. We stopped further developments here.